### PR TITLE
[10.0][ADD] Add behavior to auto-fill the automatic workflow with the payment mode (like sale order)

### PR DESCRIPTION
**Pull request by acsonefho**
Exactly as the sale order, the automatic workflow of the invoice should be automatically set depending on the payment mode.

### CODE REVIEW

**Review by Cedric-Pigeon — APPROVED** [review]

**Comment by rafaelbn** [comment]
Hi @acsonefho , please could you review why travis fail :red_circle: ? thank you!

**Comment by Cedric-Pigeon** [comment]
@rafaelbn this one is green now.

File: sale_automatic_workflow/views/account_invoice.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="account_invoice_form_view">
+        <field name="name">account.invoice.form (in sale_automatic_workflow)</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_form"/>
+        <field name="priority" eval="95"/>
+        <field name="arch" type="xml">
+            <field name="payment_mode_id" position="after">
```
Comment (rousseldenis) [inline sale_automatic_workflow/views/account_invoice.xml]: @acsonefho This should go to sale_automatic_workflow_payment_mode module

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@rafaelbn This one is green :green_apple:

**Comment by rousseldenis** [comment]
@OCA/crm-sales-marketing-maintainers

**Pull request merged**